cmd/snap-update-ns/main.go

```diff
@@ -206,3 +214,51 @@ func debugShowChanges(changes []*Change, header string) {
 		logger.Debugf("%s: (none)", header)
 	}
 }
+
+func applyUserFstab(snapName string) error {
```

Comment:
I'd love if we can find a way to make the apply function generic. Perhaps wecan just move the pre-processing code or even make that step more generic (pass a callable into the fn)